lib/node_modules/@stdlib/stats/base/range/lib/accessors.js

```diff
@@ -0,0 +1,82 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+
+
+// MAIN //
+
+/**
+* Computes the range of a strided array using the accessor protocol.
+*
+* @param {PositiveInteger} N - number of indexed elements
+* @param {Object} x - input array-like object supporting the accessor protocol
+* @param {Function} get - accessor function to retrieve array values
+* @param {integer} stride - stride length
+* @returns {number} range (max - min)
+*
+* @example
+* function accessor( arr, idx ) {
+*     return arr.get( idx );
+* }
+*
+* var x = {
+*     data: [ -2.0, 1.0, 1.0, -5.0, 2.0, -1.0 ],
+*     get: function( i ) { return this.data[ i ]; }
+* };
+*
+* var v = range( 3, x, accessor, 2 );
+* // returns 4.0
+*/
+function range( N, x, get, stride ) {
+	if ( N <= 0 ) {
+		return NaN;
+	}
+	if ( N === 1 || stride === 0 ) {
+		const v = get( x, 0 );
```

Comment:
We use ES5. All of this ES6+ usage needs to be changed. Please study the conventions in other files.